lib/x12.rb

```diff
@@ -1,6 +1,25 @@
+module X12
+  ROOT = File.expand_path('../..', __FILE__)
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.